ee/indexeddb/comparer.go

```diff
@@ -0,0 +1,33 @@
+package indexeddb
+
+import (
+	"github.com/kolide/goleveldb/leveldb/comparer"
+)
+
+// chromeComparer is a thin wrapper around the default comparer to allow us
+// to use it as `idb_cmp1`.
+type chromeComparer struct {
+	defaultComparer comparer.Comparer
+}
+
+func newChromeComparer() *chromeComparer {
```

Comment:
Is every instance of this the same / do we need to create new ones on usage? (Vs making a package variable one)